src/stats.cpp

```diff
@@ -0,0 +1,90 @@
+
```

Comment:
I suspect that this implementation is copied or adopted from somewhere else? If so, it might be good to refer to it in a comment here.

Reply:
It is free software, but you are right. Have moved the implementation of the inverse error function to a header file with appropriate license references, etc.